core/templates/pages/voiceover-admin-page/voiceover-admin-page.component.html

```diff
@@ -38,10 +38,10 @@ <h3>Update language/accent pairs available for learners and voice artists</h3>
         <mat-form-field class="oppia-language-accent-dropdown-selector">
           <mat-label>Language Accent pair</mat-label>
           <mat-select (selectionChange)="addLanguageAccentCodeSupport($event.value)">
-            <mat-option *ngFor="let languageAccentCodeToDescription of availableLanguageAccentCodesToDescriptions | keyvalue"
```

Comment:
Swapped the variable to store `language accent description to language code` instead of `language accent code to language accent description`. 
Reason: We need a sorted list of available language accents in the dropdown. JS always sort the keys in a dict hence reversed the key value pairs to get sorted language accent descriptions.